### ISSUE TITLE
Packages occasionally not recognized as AUR packages

### ISSUE THREAD

**Issue by tlunter** [fosskers/aura#182]
```
greyjoy I ∴ sudo aura -A xmobar-git
auraaura >>= The following are not AUR packages:
xmobar-git

aura >>= Determining dependencies...
aura >>= Continue? [Y/n] 
greyjoy I ∴ sudo aura -A xmobar-git
auraaura >>= Determining dependencies...
aura >>= AUR Packages:
xmobar-git

aura >>= Continue? [Y/n]
```

The first time I try to install `xmobar-git` it says it's not an AUR package, the second time it works just fine.

**Comment by abougouffa** [comment]
I have the same bug in my Arch, when I try to run

`➜  ~  sudo aura -Aav numix-themes-archblue`
`aura >>= The following are not AUR packages:`
`numix-themes-archblue`

**Comment by rlcintra** [comment]
I can confirm this is still a bug.

    # aura -Ax libappindicator-gtk2
    aura >>= The following are not AUR packages:
    libappindicator-gtk2

    # aura -Ax libappindicator-gtk3
    aura >>= The following are not AUR packages:
    libappindicator-gtk3

Please reopen the issue.

**Comment by vmassuchetto** [comment]
Maybe here:

    $ sudo aura -A linux-pf-headers
    aura >>= The following are not AUR packages:
    linux-pf-headers

    $ wget -q "https://aur.archlinux.org/packages/linux-pf-headers/" -O - | grep "<title>"
        <title>AUR (en) - linux-pf-headers</title>

**Comment by jcelliott** [comment]
I looks like this error can also happen when an AUR package can build multiple packages. For example, aura fails to build these packages: [python2-pysqlite](https://aur.archlinux.org/packages/python2-pysqlite/) and [python2-stopit](https://aur.archlinux.org/packages/python2-stopit/).

**Comment by flamusdiu** [comment]
@jcelliott @abougouffa known issue with virtual packages. That will be autofixed when `aura 2.0` comes out.

**Comment by Robsteranium** [comment]
I'm facing this issue when trying to install [moka-icon-theme-git](https://aur.archlinux.org/packages/moka-icon-theme-git/) and [faba-mono-icons-git](https://aur.archlinux.org/packages/faba-mono-icons-git/) with Aura v1.3.2.0

**Comment by Robsteranium** [comment]
Ah this looks like it might be because aura is now using AUR4 which doesn't contain those packages.

**Comment by fosskers** [comment]
@Robsteranium Can you post the output that you're seeing?

**Comment by Robsteranium** [comment]
@fosskers I did have `The following are not AUR packages:` as in the above posts. I've just tried again and it looks like aura *can* find the two packages I mentioned. Perhaps there was a temporary issue with the network or migration to aur4 etc.

Whatever the problem, this is fixed for me now. Thanks.

**Comment by amosonn** [comment]
This didn't work for me at all with aura-bin 1.3.0.5-1, not even after multiple times:
```
[~]% sudo aura -Aux        
aura >>= Fetching package information...
aura >>= Comparing package versions...
aura >>= AUR Packages to upgrade:
aura-bin : 1.3.0.5-1 => 1.3.2.0-1
gphotofs : 0.5-1 => 0.5-3
libmysqlclient : 5.6.24-1 => 5.6.26-1
mysql : 5.6.24-1 => 5.6.26-1
mysql-clients : 5.6.24-1 => 5.6.26-1
python2-pyalsaaudio : 0.7-1 => 0.8.2-1
thermald : 1.4.2-1 => 1.4.3-1
vim-nerdcommenter : 2.3.0-2 => 2.3.0-3
xflux : 20130901-2 => 20130901-3
xpdf : 3.04-3 => 3.04-4

aura >>= The following are not AUR packages:
aura-bin
gphotofs
libmysqlclient
mysql
mysql-clients
python2-pyalsaaudio
thermald
vim-nerdcommenter
xflux
xpdf

aura >>= No valid packages specified.
```
Same happened when trying a specific package, such as aura-bin itself.
Then I upgraded aura-bin manually to 1.3.2.0-1, and then things went a bit more smoothly:
```
[~]% sudo aura -Aux    
aura >>= Fetching package information...
aura >>= Comparing package versions...
aura >>= AUR Packages to upgrade:
gphotofs : 0.5-1 => 0.5-3
libmysqlclient : 5.6.24-1 => 5.6.26-1
mysql : 5.6.24-1 => 5.6.26-1
mysql-clients : 5.6.24-1 => 5.6.26-1
python2-pyalsaaudio : 0.7-1 => 0.8.2-1
thermald : 1.4.2-1 => 1.4.3-1
vim-nerdcommenter : 2.3.0-2 => 2.3.0-3
xflux : 20130901-2 => 20130901-3
xpdf : 3.04-3 => 3.04-4

aura >>= The following are not AUR packages:
libmysqlclient
mysql-clients
python2-pyalsaaudio

aura >>= Determining dependencies...
aura >>= AUR Packages:
gphotofs
mysql
thermald
vim-nerdcommenter
xflux
xpdf

aura >>= Continue? [Y/n] 
```
Turned out, that mysql-clients and libmysqlclient are both split packages inside mysql, so they were installed anyway after mysql finished building.
Similarly, python2-pyalsaaudio was a split package inside python-pyalsaaudio, which was never offered to upgrade, so it only worked manually.
Bottom line - this issue might also have something to do with mishandling of split packages, or maybe deps too.

**Comment by flamusdiu** [comment]
Yeah, split packages sometimes don't work properly which will be fixed in Aura2.

**Comment by amosonn** [comment]
Still, for some reason nothing worked before I upgraded manually to aura1.3.2.0-1.

**Comment by kgadek** [comment]
The same error shows up if you have old aura — the repo layout changed.

To fix, reinstall from AUR:

```bash

$ git clone https://aur.archlinux.org/aura-bin.git
$ cd aura-bin
$ makepkg -sir
```

**Comment by adeel** [comment]
I can confirm that the above fix by @kgadek worked for me.

**Comment by Ticki** [comment]
This is still a problem:

```
aura >>= [0;32mFetching package information...
[0maura >>= [0;32mComparing package versions...
[0maura >>= [0;32mAUR Packages to upgrade:[0m
[0;36maura-bin : 1.3.4-1 => 1.3.5-1[0m
[0;36mbroadcom-wl : 6.30.223.248-7 => 6.30.223.271-2[0m
[0;36mexa-git : 527-1 => 560-1[0m
[0;36mfreetype2-infinality : 2.6-2 => 2.6.2-1[0m
[0;36mldm : 0.6.2-2 => 0.7-2[0m
[0;36mmultirust : 0.7.0-1 => 0.8.0-2[0m
[0;36mrustfmt : 0.3.0-1 => 0.4.0-1[0m
[0;36mthermald : 1.4.3-1 => 1.5.3-1[0m

aura >>= [0;31mThe following are not AUR packages:[0m
[0;36maura-bin[0m
[0;36mbroadcom-wl[0m
[0;36mexa-git[0m
[0;36mfreetype2-infinality[0m
[0;36mldm[0m
[0;36mmultirust[0m
[0;36mrustfmt[0m
[0;36mthermald[0m

aura >>= [0;31mNo valid packages specified.[0m
```

**Comment by lucamolteni** [comment]
Same, reinstalling Aura from aura-bin as @kgadek said unstucked those upgrades. So I guess the problem is that Aura never gets upgraded?

**Comment by fosskers** [comment]
Yeah, when something fundamental breaks and aura can't even update itself it's a huge pain. Manually moving to `1.3.5` should clear things up, though.
